proto/hypertrace/agent/config/v1/config.proto

```diff
@@ -1,11 +1,11 @@
 syntax = "proto3";
 
-package org.hypertrace.agent.config;
+package hypertrace.agent.config.v1;
 
 import "google/protobuf/wrappers.proto";
 
-option go_package = "github.com/hypertrace/goagent/config";
-option java_package = "org.hypertrace.agent.config";
+option go_package = "github.com/hypertrace/agent-config/v1";
```

Comment:
I'm not sure about flattening the packages this much, as we will soon put other proto in here - specifically the grpc definitions. Why not keep the packages consistent?

`github.com/hypertrace/agent-config/hypertrace/agent/config/v1`
so then we can use 
`github.com/hypertrace/agent-config/hypertrace/agent/api/v1`
for the grpc definitions

Reply:
Good point. I think having `github.com/hypertrace/agent-config/hypertrace/agent` is redundant, how about now?